Features/Context/SubContext/Authentication.php

```diff
@@ -53,7 +53,7 @@ public function goToPlatformUiAndLogIn($username, $password)
      * @Given I am logged in as admin on PlatformUI
      * @Given I am logged in as an Editor in PlatformUI
```

Comment:
@andrerom: Indeed.

The `$user` and `$password` are for the default user, which is admin. ref: https://github.com/ezsystems/PlatformUIBundle/blob/loggedAsAdminFIx/Features/Context/PlatformUI.php#L40
(this can be modified in configuration).

Naming the function `loggedAsUser...` seems more inline with its implementation (IMHO), but what if the default user is not an Admin? (perhaps document it?)

Regarding the editor, I'm not certain why it's using this function, but seems related to current login issues with non-admin user (ping @miguelcleverti)

In any case, once it's sorted, the sentence could probably use either a pre-existing user or, create a new user with 'Editor' credentials. see https://github.com/ezsystems/BehatBundle/blob/master/Context/EzContext.php#L248